server/cmd/mmctl/commands/command.go

```diff
@@ -188,7 +188,9 @@ func createCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error
 func listCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	var teams []*model.Team
 	if len(args) < 1 {
-		teamList, _, err := c.GetAllTeams(context.TODO(), "", 0, 10000)
+		teamList, err := getPages(func(page, numPerPage int, etag string) ([]*model.Team, *model.Response, error) {
+			return c.GetAllTeams(context.TODO(), etag, page, numPerPage)
+		}, 250)
```

Comment:
Other places in the code use 200 instead of 250. Let's stick with that number for consistency.

Reply:
Sorry for oversight. Will fix that!